Add Yu-Scene

ericlay opened the pull request:
Add Yu-Scene tracker. Tested and working. 

Attempted to optimize the image.

Purfview commented:
>Attempted to optimize the image

That's a joke, no? 😛
Nevermind, I'll optimize it. Please add Clouflare...

Purfview commented:
And add it before "Yubraca" so it would be in abc logic.

ericlay commented:
Sorry didn't know which way to go with the dash. Moved it up for alpha order, added the cloudflare bits and, I did find the optimization guide under issues and replaced the image with something that should be better, hopefully
